README.md

```diff
@@ -12,6 +12,7 @@
 
 ## Getting Started
 
+use this link to reach my portfolio.
 sameh080081.github.io
```

Comment:
- [ ] Kindly make this link click able so that users don't need to copy the link and paste in the browser, please use this format 👍🏿` [live demo](https://sameh080081.github.io/)`